lte/gateway/c/oai/lib/openflow/controller/GTPApplication.cpp

```diff
@@ -116,6 +117,14 @@ void GTPApplication::add_uplink_tunnel_flow(
   of13::SetFieldAction set_eth_dst(new of13::EthDst(uplink_port));
   apply_ul_inst.add_action(set_eth_dst);
 
+  if (ev.get_ue_info().get_vlan() > 0) {
+    of13::PushVLANAction push_vlan(0x8100);
+    apply_ul_inst.add_action(push_vlan);
+
+    uint16_t vid = OFPVID_PRESENT | ev.get_ue_info().get_vlan();
+    of13::SetFieldAction set_vlan(new of13::VLANVid(vid));
+    apply_ul_inst.add_action(set_vlan);
+  }
```

Comment:
accessing vlan info more than once, save it in a local variable first and use the local variable for access.

Reply:
ok.